projects/smarti18n/src/lib/services/loaders/http-locale-loader.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import { ObjMap } from '../../models';
+import { ConfigService } from '../config/config.service';
+import { LocaleLoaderService } from './locale-loader.service';
+import { consumeBinding } from '../../../../../../node_modules/@angular/core/src/render3/instructions';
+
+@Injectable()
+export class HttpLocaleLoaderService extends LocaleLoaderService {
+	/**
+	 * Creates an instance of HttpLoaderService.
+	 * @param http
+	 * @memberof HttpLoaderService
+	 */
+	constructor(
+		private http: HttpClient,
+		private configService: ConfigService
+	) {
+		super();
+	}
+
+
+	/**
+	 * Returns an observable for the loading of a locale
+	 * @param locale The desired locale.
+	 * @returns The result of the ```.i18n.json``` file.
+	 * @memberof HttpLoaderService
+	 */
+	public load(locale: string): Observable<ObjMap<string>> {
+		if (!this.configService.config.loader)
+			throw new Error('HttpLoaderService needs a `loaderConfig` set');

```

Comment:
I guess it would be nice to improve the error message. There is no method  "loaderConfig"... I guess it would be better to reference the setConfig object. Also, backticks have no effect here =P

Reply:
they are not supposed to have an effect, just being a different quotation than string